### PR TITLE
Button Group

### PR DESCRIPTION
**Pull request by ShawnToubeau**
This PR adds the ability to group button components together

### CODE REVIEW

File: src/app/components/Common/Button/index.tsx [hunk 1]
```diff
@@ -10,14 +10,16 @@ export interface IButtonProps {
...
 }
 
 class Button extends React.Component<IButtonProps> {
   public static defaultProps = {
     size: "medium",
     color: "primary",
     outline: false,
-    radius: "large"
+    radius: "large",
+    className: null
```
Comment (uptonm) [inline src/app/components/Common/Button/index.tsx]: set this to "" because in the DOM it will read 
class="btn btn--blah" vs class="btn btn--blah null"

**Review by uptonm — APPROVED** [review]
Just left one comment for a quick fix then its good to merge

File: src/app/components/Common/Button/index.tsx (hunk 1 above)
Comment (ShawnToubeau) [reply src/app/components/Common/Button/index.tsx]: good catch

**Review by uptonm — APPROVED** [review]
Looks good, just run `loki:test` and `loki:commit` and push the new screenshots, then ur good to merge

**Pull request merged**